### PR TITLE
Update on Hummingbird situation

### PR DESCRIPTION
**Pull request by kpodemski**

### CODE REVIEW

**Review by SharakPL — APPROVED** [review]

**Review by PrestaEdit — CHANGES_REQUESTED** [review]
As it's an edit, I will appreciate that we leave the first content and just add the newest notice, there.

**Review by matthieu-rolland — APPROVED** [review]

**Comment by matthieu-rolland** [comment]
> As it's an edit, I will appreciate that we leave the first content and just add the newest notice, there.

I think `While we would love to see Hummingbird in PrestaShop 9` reminds the reader that's that indeed what we want, or you want to make it more obvious that there is potentially a "change of plan" ?

**Comment by PrestaEdit** [comment]
More obious, and I'd like to get the updated infos in addition of previous read, as an update and not a replacement, when I read a blog post from past, on my side.

**Comment by kpodemski** [comment]
@PrestaEdit I edited the text, I hope we met halfway to clarify the message. The main point is to state the current situation.

**Comment by kpodemski** [comment]
cc @matks @MatShir

**Comment by PrestaEdit** [comment]
> @PrestaEdit I edited the text, I hope we met halfway to clarify the message. The main point is to state the current situation.

Thanks @kpodemski ! I appreciate the change, and approve it ! :)

**Review by PrestaEdit — APPROVED** [review]

File: content/news/posts/2022/new-theme-announce.md [hunk 1]
```diff
@@ -9,8 +9,10 @@ icon: icon-laptop
 tags: [news, theme, "8.x"]
 ---
 
-{{% notice type="success" title="July 2023: Hummingbird is coming to PrestaShop 9!" icon="megpahone" %}}
- It has been decided that Hummingbird will be the default theme for PrestaShop 9. You can take an active part in developing this new theme by [joining the discussion on Slack](https://www.prestashop-project.org/slack/), testing the theme, and submitting improvements. Make sure to check out the [theme's repository on GitHub](https://github.com/PrestaShop/hummingbird).
+{{% notice type="warning" title="April 2024: Hummingbird situation" icon="megpahone" %}}
```
Comment (matks) [inline content/news/posts/2022/new-theme-announce.md]:
```suggestion
{{% notice type="warning" title="April 2024: Hummingbird situation" icon="megaphone" %}}
```

**Review by matks — APPROVED** [review]
Good for me, let's publish tomorrow. Also added a little fix

**Pull request merged**